Creating metric card for MAE (#4252)
huggingface/datasets: metrics/mae/README.md

## How to Use

At minimum, this metric requires predictions and references as inputs.

```python
>>> mae_metric = datasets.load_metric("mae")
>>> predictions = [2.5, 0.0, 2, 8]
>>> references = [3, -0.5, 2, 7]
>>> results = mae_metric.compute(predictions=predictions, references=references)
```

### Inputs

Mandatory inputs: 
- **predictions**: array-like of shape (`n_samples,`) or (`n_samples`, `n_outputs`), representing the estimated target values.
- **references**: array-like of shape (`n_samples,`) or (`n_samples`, `n_outputs`), representing the ground truth (correct) target values.

Optional arguments:
- **sample_weight**: array-like of shape (`n_samples,`) representing sample weights. The default is `None`.
- **multioutput**: `raw_values`, `uniform_average` or array-like of shape (`n_outputs,`), which defines the aggregation of multiple output values; the array-like value defines weights used to average errors. `raw_values` returns a full set of errors in case of multioutput input, whereas `uniform_average` means that the errors of all outputs are averaged with uniform weight. The default value is `uniform_average`.

### Output Values

> - **mae**(`float`): This metric outputs a dictionary, containing the mean absolute error score. 
> - 
> - If multioutput is `raw_values`, then the score is returned for each output separately. If multioutput is `uniform_average` or an ndarray of weights, then the weighted average of all output errors is returned. It ranges from `0.0` to `1.0`, with the best value is 0.0.

Reviewer: ```suggestion
This metric outputs a dictionary, containing the mean absolute error score, which is of type:
- `float`: if multioutput is `uniform_average` or an ndarray of weights, then the weighted average of all output errors is returned.
- numeric array-like of shape (`n_outputs,`): if multioutput is `raw_values`, then the score is returned for each output separately. 

Each MAE `float` value ranges from `0.0` to `1.0`, with the best value being 0.0.
```

Output Example(s):
```python

> {'mae': array([0.5, 1. ])}

Reviewer: Reverse order, as the first one is the default behavior:
```suggestion
{'mae': 0.5}
```

```

> ```python

Reviewer: ```suggestion

If `multioutput="raw_values"`:
```python
```

> {'mae': 0.5}

Reviewer: ```suggestion
{'mae': array([0.5, 1. ])}
```
